UPDATE.md

```diff
@@ -16,6 +16,63 @@ the "Configuration updates" section of this file.
 
 ## Updating Lightning
 
+### Summary
+For a typical site that has a properly configured directory for exporting config
+that is managed by your VCS, you would generally follow these steps:
+
+#### In your development or local environment.
+1. Read the [release notes](https://github.com/acquia/lightning/releases)
+   for the release to which you are updating, and any other releases between
+   your current version.
+   
+1. Update your codebase.
+  
+  ```
+  composer self-update
+  composer require acquia/lightning:~3.1.0 --no-update
```

Comment:
How are we going to remember to change this version constraint in future minor releases?